frontend/src/static/js/components/test/webstatus-stats-global-feature-count-chart-panel.test.ts

```diff
@@ -38,11 +38,11 @@ describe('WebstatusStatsGlobalFeatureCountChartPanel', () => {
     apiClientStub = stub(new APIClient(''));
     fetchAndAggregateDataStub = stub(
       WebstatusLineChartPanel.prototype,
-      '_fetchAndAggregateData',
+      '_populateDataForChart',
     );
     setDisplayDataFromMapStub = stub(
```

Comment:
```suggestion
    processDisplayDataFromMapStub = stub(
```

Reply:
You'll need to change the uses of setDisplayDataFromMapStub to processDisplayDataFromMapStub

Reply:
I've updated these var names in their respective files.